src/WebJobs.Script.NuGet/WebJobs.Script.nuspec

```diff
@@ -27,7 +27,7 @@
       <dependency id="Edge.js" version="6.5.1" />
       <dependency id="Microsoft.CodeAnalysis.CSharp.Scripting" version="1.3.2" />
       <dependency id="Microsoft.AspNet.WebApi.Core" version="5.2.3" />
-      <dependency id="FSharp.Compiler.Service" version="8.0.0" />
+      <dependency id="FSharp.Compiler.Service" version="9.0.1" />
```

Comment:
Only place we're revving to a new major version - rest are all minor version. F# is still in preview, and it's unlikely there are any breaking changes here.